test_schemas/en/test_individual_response_hub_disabled.json

```diff
@@ -0,0 +1,418 @@
+{
```

Comment:
Can the name be `test_individual_response_on_hub_disabled` as i thought it was IR but hub is disabled, not that IR on the hub is disabled.

Reply:
Also updated in the validator repo